app/calculation/index.js

```diff
@@ -7,10 +7,10 @@ const schemeYears = require('./scheme-years')
 function calculate (value) {
   const schemeYearValues = !isNaN(value)
     ? schemeYears.map(schemeYear => ({ schemeYear, value }))
-    : [{ schemeYear: 2021, value: value.value2021 || 0 },
-        { schemeYear: 2022, value: value.value2022 || 0 },
-        { schemeYear: 2023, value: value.value2023 || 0 },
-        { schemeYear: 2024, value: value.value2024 || 0 }]
+    : [{ schemeYear: 2021, value: value.value2021 || 0 }, // eslint-disable-line
+      { schemeYear: 2022, value: value.value2022 || 0 }, // eslint-disable-line
+      { schemeYear: 2023, value: value.value2023 || 0 }, // eslint-disable-line
+      { schemeYear: 2024, value: value.value2024 || 0 }] // eslint-disable-line
```

Comment:
Latest version of StandardJS is struggling to evaluate this line consistently.  Sometimes believe indentation should be 6 spaces and sometimes 8.

Probably a sign that we should simplify this and not have ternary operator.  Have left for now as changing calculation code is outside scope of WebPack update.